spec/index.bs

```diff
@@ -513,6 +508,29 @@ interface mixin GPUObjectBase {
         An internal slot holding the [=device=] which owns the [=internal object=].
 </dl>
 
+{{GPUObjectDescriptorBase}} has the following members:
+
+<dl dfn-type=dict-member dfn-for=GPUObjectDescriptorBase>
+    : <dfn>label</dfn>
+    ::
+        The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
+</dl>
+
+
+: <dfn dfn>Debug label</dfn>
+:: A string provided for use by development tools (such as error/warning messages,
+    browser developer tools, or platform debugging utilities) to identify the underlying
+    [=internal object=], debug group, or debug marker to the developer.
+    It has no specified format, and implementations MUST NOT use label contents in heuristics.
```

Comment:
nit: not just heuristics, it must not affect behavior at all.

Reply:
nit: I think we use the "should not" wording because there's no way to enforce the "must not" behavior via tests.

Reply:
I think we can and should keep "must not" even though it's not enforceable.